### PR TITLE
MINOR: add unit test for StateStoreSerdes

### PR DESCRIPTION
**Pull request by mjsax**

### CODE REVIEW

**Comment by mjsax** [comment]
Call for reviews @bbejeck @guozhangwang @dguy

**Comment by dguy** [comment]
retest this please

File: streams/src/test/java/org/apache/kafka/streams/state/StateSerdesTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void shouldThrowIfValueClassIsNullForBuiltinTypes() {
+        StateSerdes.withBuiltinTypes("anyName", byte[].class, null);
+    }
+
+    @Test
+    public void shouldReturnSerdesForBuiltInKeyAndValueTypesForBuiltinTypes() {
+        final Class[] supportedBuildInTypes = new Class[] {
```
Comment (bbejeck) [inline streams/src/test/java/org/apache/kafka/streams/state/StateSerdesTest.java:45]: Maybe consider JUnit Parameters here, but fine as is.

EDIT: Thinking some more about this, I'd leave it as is.

**Comment by bbejeck** [comment]
retest this please

**Comment by bbejeck** [comment]
Thanks, @mjsax just one minor question comment otherwise LGTM.

**Comment by mjsax** [comment]
Updated this.

**Comment by guozhangwang** [comment]
LGTM. Merged to trunk and 1.0.

**Pull request closed without merging**